itest/rust/src/builtin_tests/geometry/transform2d_test.rs

```diff
@@ -106,3 +112,48 @@ fn transform2d_xform_equiv() {
         "operator: Transform2D * Rect2 (2)"
     );
 }
+
+#[itest]
+fn transform2d_xform_inv_equiv() {
+    let vec = Vector2::new(1.0, 2.0);
+
+    assert_eq_approx!(
+        TEST_TRANSFORM_ORTHONORMAL.xform_inv(vec),
+        vec.to_variant()
+            .evaluate(
+                &TEST_TRANSFORM_ORTHONORMAL.to_variant(),
+                VariantOperator::MULTIPLY
+            )
+            .unwrap()
+            .to::<Vector2>(),
+        "operator: Transform2D * Vector2"
+    );
```

Comment:
Please use local variable definitions instead of mega-asserts :slightly_smiling_face: 

The 
```rs
.to_variant()
            .evaluate(&input.to_variant(), VariantOperator::MULTIPLY)
            .unwrap()
            .to::<T>(),
```
is also repeated many times and could be extracted into a `multiply_variant` (or so) method.